index.js

```diff
@@ -23,6 +24,7 @@ const convertWithOptions = (document, format, filter, options, callback) => {
                     path.join(process.env['PROGRAMFILES(X86)'], 'LIBREO~1/program/soffice.exe'),
                     path.join(process.env['PROGRAMFILES(X86)'], 'LibreOffice/program/soffice.exe'),
                     path.join(process.env.PROGRAMFILES, 'LibreOffice/program/soffice.exe'),
+                    userInstallation = `file:///${installDir.name}`;
```

Comment:
wait, assign `userInstallation` variable in paths array?

Reply:
you are correct, that doesn't seem right!